packages/net/auth.pony

```diff
@@ -1,23 +1,3 @@
 primitive NetAuth
```

Comment:
I think this file should be named `net_auth.pony` (even though it is already in a `net` subdir), because the principal type declared in the file is `NetAuth`.